CMakeModules/AFconfigure_forge_dep.cmake

```diff
@@ -51,13 +51,21 @@ if(AF_BUILD_FORGE)
     set(FETCHCONTENT_UPDATES_DISCONNECTED ${af_FETCHCONTENT_UPDATES_DISCONNECTED})
     install(FILES
         $<TARGET_FILE:forge>
-        $<TARGET_RUNTIME_DLLS:forge>
         $<$<PLATFORM_ID:Linux>:$<TARGET_SONAME_FILE:forge>>
         $<$<PLATFORM_ID:Darwin>:$<TARGET_SONAME_FILE:forge>>
         $<$<PLATFORM_ID:Linux>:$<TARGET_LINKER_FILE:forge>>
         $<$<PLATFORM_ID:Darwin>:$<TARGET_LINKER_FILE:forge>>
         DESTINATION "${AF_INSTALL_LIB_DIR}"
         COMPONENT common_backend_dependencies)
+
+    if(AF_INSTALL_STANDALONE)
+        cmake_minimum_required(VERSION 3.21)
```

Comment:
Anyone who wants to do an stand alone installation would need newer cmake ? Is that the intended behaviour going forward ?

Reply:
Yeah, I think that is an acceptable requirement. I think we are the only people interested in that functionality.

Reply:
We are too @umar456  
tks.

Reply:
@WilliamTambellini Is this a difficult requirement for you? Also I don't think you are interested in the forge dependency. This is only required if you are interested in forge and want to include forge as part of the install.

Reply:
no, we are cool on it: we can just 'make install' and copy manually the libs/includes we are into.